linux/helper_suse.sh

```diff
@@ -28,7 +28,11 @@ add_repo() {
     wget -q "http://${domain}/infrastructure_agent/gpg/newrelic-infra.gpg" -O newrelic-infra.gpg
     rpm --import newrelic-infra.gpg && rm newrelic-infra.gpg
 
-    zypper -n addrepo "http://${domain}/infrastructure_agent/linux/zypp/sles/${VERSION_ID}/x86_64/newrelic-infra.repo"
+    # Pointing to cacheless endpoint if we match the prod URL. Non prod URL will not match
+    wget -q "http://${domain}/infrastructure_agent/linux/zypp/sles/${VERSION_ID}/x86_64/newrelic-infra.repo" -O "repo.tmp"
```

Comment:
instead of downloading directly the repo we download it and move it manually to the location

Reply:
wouldn't make sense to create the repo locally as it's done with rocky linux? just for consistency